operators/multiclusterobservability/controllers/multiclusterobservability/analytics/rightsizing_namespace_placementbinding_test.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+// Licensed under the Apache License 2.0
+
+package analytics
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types" // ✅ important
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+
+	policyv1 "open-cluster-management.io/governance-policy-propagator/api/v1"
+)
+
+func TestCreatePlacementBinding_CreatesWhenNotFound(t *testing.T) {
+	scheme := runtime.NewScheme()
+	_ = policyv1.AddToScheme(scheme)
+
+	client := fake.NewClientBuilder().WithScheme(scheme).Build()
+
+	err := createPlacementBinding(context.TODO(), client)
```

Comment:
nit: you can create `ctx := context.Background()` and use it in the test repeatedly.

Reply:
ACK